### PR TITLE
Create basic application which responds to hello request

### PR DESCRIPTION
**Pull request by FusTaFah**

### CODE REVIEW

File: gamebox-api/Controllers/WeatherForecastController.cs
Comment (michaelenger) [inline gamebox-api/Controllers/WeatherForecastController.cs:1]: Is this file necessary?

File: gamebox-api/Models/Hello.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace gamebox_api.Models
```
Comment (michaelenger) [inline gamebox-api/Models/Hello.cs:1]: Should we perhaps use just `GameBox` as the parent namespace? Not sure how easy/hard it is to change that everywhere... inluding the name of the VS solution 😬

File: gamebox-api/Models/Hello.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+{
+    public class Hello
```
Comment (michaelenger) [inline gamebox-api/Models/Hello.cs:3]: I think I'd prefer for this model to be called `HealthStatus` or something generic like that.

File: gamebox-api/Controllers/WeatherForecastController.cs
Comment (FusTaFah) [reply gamebox-api/Controllers/WeatherForecastController.cs:1]: ah, that came with the project. no it's not necessary i'll delete it.

File: gamebox-api/Models/Hello.cs (hunk 1 above)
Comment (FusTaFah) [reply gamebox-api/Models/Hello.cs:1]: can do!

File: gamebox-api/Models/Hello.cs (hunk 2 above)
Comment (FusTaFah) [reply gamebox-api/Models/Hello.cs:3]: I'll change the name

File: gamebox.sln
Comment (michaelenger) [inline gamebox.sln:1]: Would it be possible to rename the solution to `GameBox` including the capitalisation? 😇 
As well as the namespace?

Seems more conformant to C# standards.

Comment (FusTaFah) [reply gamebox.sln:1]: can do!

**Review by michaelenger — APPROVED** [review]
